### ISSUE TITLE
Variable Explorer doesn't show simple array view on Ubuntu 18.04

### ISSUE THREAD

**Issue by tdalis** [spyder-ide/spyder#7283]
I have a Variable Exporer problem, I try to see the arrays in the form of a table when I double click on the Name of the Variable in Variable explorer but I get numpy.ndarray instead of numpy.array - Do I do something wrong in my settings? Do I need to reinstall both Anaconda and Spyder? I am using Ubuntu 18.04 LTS and Spyder version 3.2.6+dfsg1-2 - Python version is 3.6
![screenshot from 2018-06-12 13-05-13](https://user-images.githubusercontent.com/19262536/41289437-a69f8dec-6e41-11e8-821d-03a2dfe94346.png)

**Comment by ccordoba12** [comment]
Please post here the code you have in the Editor that generates the Dataframe you're trying to view through our Variable Explorer.

**Comment by tdalis** [comment]
`#Import libraries
import numpy as np
import matplotlib.pyplot as plt
import pandas as pd

#Import dataset
df = pd.read_csv('Data.csv')
X = df.iloc[:, :-1].values
Y = df.iloc[:, 3].values

#Taking care of missing data
from sklearn.preprocessing import Imputer
imputer = Imputer(missing_values='NaN', strategy = 'mean', axis = 0)
imputer = imputer.fit(X[:, 1:3])
X[:, 1:3] = imputer.transform(X[:, 1:3])

# Encoding categorical data
from sklearn.preprocessing import LabelEncoder, OneHotEncoder
labelencoder_x = LabelEncoder()
X[:, 0] = labelencoder_x.fit_transform(X[:, 0])
onehotencoder = OneHotEncoder(categorical_features = [0])
X = onehotencoder.fit_transform(X).toarray()
labelencoder_y = LabelEncoder()
Y = labelencoder_y.fit_transform(Y)

# Spliting dataset into training and Test set
from sklearn.cross_validation import train_test_split
X_train, X_test, Y_train, Y_test = train_test_split(X, Y, test_size = 0.2, random_state = 0)

# Feature Scaling
from sklearn.preprocessing import StandardScaler
sc_X = StandardScaler()
X_train = sc_X.fit_transform(X_train)
X_test = sc_X.transform(X_test)`

**Comment by ccordoba12** [comment]
Thanks @tdalis! We'll take a look at this in a future release.

**Comment by CAM-Gerlach** [comment]
Hey @tdalis , can you provide a minimal example of a ``Data.csv`` file that produces this error?

**Comment by tdalis** [comment]
it's a very simple file with four columns, two categorical and two numerical

**Comment by tdalis** [comment]
@ccordoba12
Can you let me know when the new update is out?

**Comment by CAM-Gerlach** [comment]
We'll announce it here, on our Blog, Google Group, Facebook, Twitter, OpenCollective, etc. However, it might be some time since this is scheduled for 3.3.1 and the next update is 3.3.0.

**Comment by jondo** [comment]
@tdalis, is this still a problem for you?

**Comment by ccordoba12** [comment]
Closing because this is too old and probably it doesn't apply anymore.
